### DISCUSSION TITLE
I have issue voting for the Gyroscope Protocol

### DISCUSSION THREAD

**Discussion by wj0102307** [snapshot-labs/snapshot#927] [Q&A] [answered]
I have completed level 2 tasks and purchased NFT. I created my account, joined your family, and commented, but I can’t vote. I don't know where this is going wrong!
this is my address

0xaB888291F4127352B655fd476F64AC2ebfb8fe76

**Comment by samwhosung** [comment]
@wj0102307 which space are you trying to vote in? Have you tried to reach out the the project that setup the proposal?

**Reply by wj0102307** [reply]
GGWP-1 DAO Bootstrapping

Sir, I’m participating in this event to vote, but I can’t vote, I don’t know where the problem is

**Comment by gaosong8801** [comment]
You should be able to vote after completing level 2 before the release of level 3 tasks.

**Comment by gaosong8801** [comment]
If you get a vote, you must complete level 2 before level 3. It is not allowed if the time is later.

**Reply by wj0102307** [reply]
Then I can’t vote, does that mean I don’t have a chance to advance in the future?

**Comment by gaosong8801** [comment]
There may be opportunities in the future, but I am not very clear.

[image: image.png]


Allen ***@***.***> 于2021年10月22日周五 下午10:46写道：

**Reply by wj0102307** [reply]
Thank you very much

**Comment by metrox-eth** [comment, marked as answer]
On snapshot you can only vote if you hold the governance token in your wallet before the date of the proposal creation "the snapshot". In the case Gyroscope protocol they ask you to purchase a NFT to participate in their gamified "governance" proposal. Snapshot is not involved in Gyroscope project and advise to fully understand the conditions before purchasing governance tokens.
